From 167884ebbd336834b9b524d2b57c6ce8dc562c20 Mon Sep 17 00:00:00 2001
From: Steve Richardson <tangentaudio@gmail.com>
Date: Mon, 12 Mar 2018 18:10:48 -0700
Subject: [PATCH] alsa params

---
 src/plugins/alsa/qalsaaudiooutput.cpp | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/src/plugins/alsa/qalsaaudiooutput.cpp b/src/plugins/alsa/qalsaaudiooutput.cpp
index cd97ae8..c258ba3 100644
--- a/src/plugins/alsa/qalsaaudiooutput.cpp
+++ b/src/plugins/alsa/qalsaaudiooutput.cpp
@@ -53,10 +53,11 @@
 #include <QtMultimedia/private/qaudiohelpers_p.h>
 #include "qalsaaudiooutput.h"
 #include "qalsaaudiodeviceinfo.h"
+#include <stdlib.h>
 
 QT_BEGIN_NAMESPACE
 
-//#define DEBUG_AUDIO 1
+#define DEBUG_AUDIO 1
 
 QAlsaAudioOutput::QAlsaAudioOutput(const QByteArray &device)
 {
@@ -68,8 +69,8 @@ QAlsaAudioOutput::QAlsaAudioOutput(const QByteArray &device)
     period_frames = 0;
     buffer_size = 0;
     period_size = 0;
-    buffer_time = 100000;
-    period_time = 20000;
+    buffer_time = 200000;
+    period_time = 40000;
     totalTimeValue = 0;
     intervalTime = 1000;
     audioBuffer = 0;
-- 
2.7.4

