--- a/src/plugins/alsa/qalsaaudiooutput.cpp
+++ b/src/plugins/alsa/qalsaaudiooutput.cpp
@@ -53,10 +53,11 @@
 #include <QtMultimedia/private/qaudiohelpers_p.h>
 #include "qalsaaudiooutput.h"
 #include "qalsaaudiodeviceinfo.h"
+#include <stdlib.h>
 
 QT_BEGIN_NAMESPACE
 
-//#define DEBUG_AUDIO 1
+#define DEBUG_AUDIO 1
 
 QAlsaAudioOutput::QAlsaAudioOutput(const QByteArray &device)
 {
@@ -68,8 +69,8 @@ QAlsaAudioOutput::QAlsaAudioOutput(const QByteArray &device)
     period_frames = 0;
     buffer_size = 0;
     period_size = 0;
-    buffer_time = 100000;
-    period_time = 20000;
+    buffer_time = 500000;
+    period_time = 83333;
     totalTimeValue = 0;
     intervalTime = 1000;
     audioBuffer = 0;
-- 
2.7.4

